server/src/admin/admin.controller.ts

```diff
@@ -273,6 +274,23 @@ export const resetAllProfessorsDb = async ({ auth }: VerifyAdminType) => {
   return result;
 };
 
+/**
+ * Updates all subjects in the database to represent their full subject names
+ * @param {Auth} auth: Object that represents the authentication of a request being passed in.
+ * @returns true if operation was successful, false if operations was not successful, null if token not admin
+ */
+export const updateSubjectsDb = async ({ auth }: VerifyAdminType) => {
```

Comment:
Nit picky on my end but would like `updateDatabaseCourseFullSubjectName` be a better name? Since it is the main function